spec/models/chargeback_rate_detail_spec.rb

```diff
@@ -9,6 +9,16 @@
     end
   end
 
+  it "#find_rate" do
+    cvalue   = 50.0
+    cbt1 = FactoryGirl.build(:chargeback_tier, :start => 0, :finish => 10, :fixed_rate => 3.0, :variable_rate => 0.3)
```

Comment:
there should be used:
let(:cbt2) { FactoryGirl.build(:chargeback_tier, :start => 10, :finish => 50, :fixed_rate => 2.0, :variable_rate => 0.2) }

but I guess it is ok in this PR, because it is all places in this file.
So we can do such refactoring in follow up PR.